src/models/Model.js

```diff
@@ -1,8 +1,19 @@
+import { observable, action } from "mobx";
+import axios from 'axios';
 
-import { observable } from "mobx";
+import { CORS_ALLOW_URL, BASE_URL } from "../config/api_config";
 
-class Model {
+export class Model {
     @observable isInitialized = false;
-}
 
-export class {Model};
\ No newline at end of file
+    @action
+    getAPIBaseURL = `${CORS_ALLOW_URL}${BASE_URL}`;
```

Comment:
Why this is an action? Actions should be used when state is modified. I also think that it should have a name `ApiBaseURL` because `get` suggests a method and this is just a combination of two strings. But this is only my suggestion.